### DISCUSSION TITLE
Users don't know about this plugin as it is not "official"

### DISCUSSION THREAD

**Discussion by wrooot** [igniterealtime/openfire-pade-plugin#185] [General]
On plugin page and in Admin Console only old Openfire Meetings is provided. I know that there are blog posts about new releases, but most admins don't read them and only see what is on Plugins page in console. What about adding it to plugins list? Also, in instructions it says about pade.jar, but i see ofmeet.jar included in downloads. Is it still required and what it is used for?

**Comment by wrooot** [comment]
Maybe @guusdk  and @akrherz also want to discuss this.

**Reply by akrherz** [reply]
I don't have a preference.

**Comment by wrooot** [comment]
After installing both files they are named Pade Server and Pade Meetings in Admin Console. So i guess latter one is required to get the old Meetings tab and to have some sort of meetings scheduling. It has different naming on GitHub in the names of plugins. Personally i would make names same everywhere and if possible would combine it into one file. And of course making it visible on plugins list would make more people to find out and use it.

**Reply by deleolajide** [reply]
See https://github.com/igniterealtime/openfire-pade-plugin/issues/154

**Comment by gjaekel** [comment]
Yes, peeking different information over the timeline might be confusing for newbies:
* There is still old (, outdated) stuff in the plugin repository (OpenFire Focus Provider, Openfire Websockets?)
* There is the new OFmeet stuff in the Pàdé project
  * Here, there have been three separate plugin for the components (`offocus`, `ofmeet` and `pade`)
  * Then, `offocus` plugin have been merged into `ofmeet`
  * Upcoming, also this `ofmeet` bundle have been merged into `pade`: This will avoid problems with the startup order and "URL visibility".

**Comment by deleolajide** [comment]
Newbies should learn to read change-log files

![image](https://user-images.githubusercontent.com/110731/103346939-6b8a4500-4a8d-11eb-902d-ccb0968b3cc9.png)

**Reply by gjaekel** [reply]
But we all know: Nearly everybody read documentation -- but just as a last resort.

**Comment by deleolajide** [comment]
>Users don't know about this plugin as it is not "official"

For anyone curious about the motivation, strategy and roadmap for Pade, here is the presentation I made at FOSDEM in the name of **Ignite Realtime Community**

[![Pade presentation](https://img.youtube.com/vi/A2urDSv_2-0/0.jpg)](https://www.youtube.com/watch?v=A2urDSv_2-0&t=65s)

**Reply by gjaekel** [reply]
The Video seems to be premature interrupted at 20min.

**Reply by deleolajide** [reply]
> The Video seems to be premature interrupted at 20min.

Yes. My time management is as bad as my presentation skills. Each speaker had exactly 20 mins and was warned well in advance. Like a newbie, I got carried away :-)

**Reply by deleolajide** [reply]
Pade now builds to archiva like all official plugins

https://igniterealtime.org/projects/openfire/plugin-archive.jsp?plugin=pade
